pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -9,6 +9,11 @@ from numpy cimport (
 )
 
 
+cdef enum Exact:
+    INEXACT_MATH
+    EXACT_MATCH
+    NO_MATCH
```

Comment:
Is there a way to reuse the `Exact` enum defined in `np_datetime_strings.h`?

I tried replacing this with
```python
cdef extern from "src/datetime/np_datetime_strings.h":
    cdef enum Exact:
        INEXACT_MATCH
        EXACT_MATCH
        NO_MATCH
```
but then, during compiling, got lots of mysterious errors such as
```
In file included from pandas/_libs/index.c:798:
./pandas/_libs/tslibs/src/datetime/np_datetime_strings.h:100:29: error: unknown type name ‘pandas_timedeltastruct’; did you mean ‘npy_timedeltastruct’?
  100 | int make_iso_8601_timedelta(pandas_timedeltastruct *tds, char *outstr,
      |                             ^~~~~~~~~~~~~~~~~~~~~~
      |                             npy_timedeltastruct
```
which I couldn't make sense of

Reply:
got it, just needed to move the definition to after `pandas_timedeltastruct `